Src/BootCamp.Chapter/Program.cs

```diff
@@ -4,9 +4,26 @@ namespace BootCamp.Chapter
 {
     class Program
     {
-        static void Main(string[] args)
+        static void Main(string[] args) 
         {
-            Console.WriteLine("Hello World!");
+            string firstName = "Mert Efe";
```

Comment:
PLease use console to read input. Do not hardcode input, the point was not that.

Reply:
Hi kai I did made the necessary adjustments. Hope they are ok now.